README.md

```diff
@@ -13,27 +13,24 @@ path                  | description
 :---------------------|:-----------------------------
 `doc`                 | Contains Sphinx package documentation for the dataset. This documentation may be linked to from other packages, such as `ap_verify`.
 `raw`                 | To be populated with raw data. Data files do not need to follow a specific subdirectory structure. Currently contains a single small fits file (taken from `obs_test`) to test `git-lfs` functionality.
-`calib`               | To be populated with master calibs. Calibration files do not need to follow a specific subdirectory structure. Currently empty.
-`config`              | To be populated with dataset-specific configs. Currently contains an example file corresponding to the contents of `raw` and `refcats`.
-`templates`           | To be populated with `TemplateCoadd` images produced by a compatible version of the LSST pipelines. Must be organized as a filesystem-based Butler repo. Currently empty.
-`repo`                | A template for a Gen 2 Butler raw data repository. This directory must never be written to; instead, it should be copied to a separate location, and data ingested into the copy (this is handled automatically by `ap_verify`, see below). Note that the `_mapper` file will require updating for other instruments.
-`preloaded`           | To be populated with a Gen 3 Butler repository (see below). This repository must never be written to; instead, it should be copied to a separate location (this is handled automatically by `ap_verify`, see below). Currently empty, but can be populated from the other directories using `scripts/add_gen3_repo.py`.
-`refcats`             | To be populated with tarball(s) of HTM shards from relevant reference catalogs. Currently contains a small (useless) example tarball.
+`config`              | To be populated with dataset-specific configs. Currently empty.
+`pipelines`           | To be populated with dataset-specific pipelines. Currently contains three example files specialized for ImSim data.
+`preloaded`           | To be populated with a Gen 3 Butler repository (see below). This repository must never be written to; instead, it should be copied to a separate location (this is handled automatically by `ap_verify`, see below).
 `dataIds.list`        | List of dataIds in this repo. For use in running Tasks. Currently set to run all Ids.
 
 
 Gen 3 Collections
 -----------------
 
-The Gen 3 repository in `preloaded/` must contain the following collections.
-These may duplicate the directories above to support both Gen 2 and Gen 3 processing.
+The Gen 3 repository in `preloaded/` must contain the following collections; these may be chained collections wrapping nonstandard names.
```

Comment:
"wrapping nonstandard names"? Do you mean "these may be chained collections containing collections with other names" (or something like that)?

Reply:
Something like that. Basically, a rule saying that runs must follow any given naming scheme would make it almost impossible to create or update such a repository in the first place.

Reply:
I'm not sure if there's a way to reword it?

Reply:
"chained collections containing arbitrarily-named runs"?

Reply:
Oh, yeah, I like that! Thank you.